logging/tests/unit/handlers/test__helpers.py

```diff
@@ -0,0 +1,122 @@
+# Copyright 2017 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+
+
+class TestFlaskTrace(unittest.TestCase):
+
+    def create_app(self):
+        import flask
+
+        app = flask.Flask(__name__)
+
+        @app.route('/')
+        def index():
+            return 'test flask trace'
+
+        return app
+
+    def setUp(self):
+        self.app = self.create_app()
```

Comment:
It's fine to modify `self` in `setUp` (and `tearDown`) as she has done. If she was doing it from an actual test, that would be an issue.